lib/test.py

```diff
@@ -242,7 +247,7 @@ def run(self, server):
               self.is_equal_result and
               self.is_valgrind_clean):
             short_status = 'pass'
-            color_stdout("[ pass ]\n", schema='test_pass')
+            color_stdout("[ pass ] [ {} ]\n" . format(test_timing), schema='test_pass')
```

Comment:
There was agreement to don't make test-run output too wide to allow split the screen vertically on a laptop.

Another point here is that we're interested in large timings, not all. It would be nice to see a list of longest tests after end of testing. It seems, it is done in PR #298, which actually should resolve #286 (BTW, I updated the description of the task). Does PR #298 supersedes this one?